src/GDFactory.cpp

```diff
@@ -107,6 +108,35 @@ void GDFactory::dispatchDBEvent(const std::string& type, EventObject *value)
     p_owner->dispatch_snd_event(String(type.c_str()), value);
 }
 
+
+Armature *GDFactory::_buildChildArmature(const BuildArmaturePackage *dataPackage, Slot *slot, DisplayData *displayData) const {
```

Comment:
the meat; got a good chunk of this code from JS library. The godot specific implementation was all me though :)